Makefile

```diff
@@ -1,31 +1,35 @@
 help:                                                                           ## shows this help
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_\-\.]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
+vendor: composer.json composer.lock
```

Comment:
wouldn't it be sufficient to just look at the lock file?

Reply:
yeah, probably, since a change to composer.json without composer.lock would need a `composer update` anyways, feel free to adjust as needed